### PR TITLE
Docs: Browser cookies

### PR DESCRIPTION
**Pull request by inancgumus**
Related: grafana/xk6-browser#6.

Updates: #1309

### CODE REVIEW

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/cookie.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Cookie"
+excerpt: "Browser module: Cookie Class"
+---
+
+Cookie class representing a cookie in the [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext).
+
+| Property | Type   | Description                                                                                                                                                                          |
+| -------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```
Comment (ka3de) [inline src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/cookie.md:9]: I see many of the rows in this table reference a default value. Should we consider using a table with a `Default` column as it is used in other parts of the documentation?

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/addCookies.md [hunk 2]
```diff
@@ -3,7 +3,14 @@ title: 'addCookies()'
 excerpt: 'Clears context cookies.'
 ---
 
-Adds cookies into the `BrowserContext`. All pages within this context will have these cookies installed.
+Adds a list of [cookies](/javascript-api/k6-experimental/browser/browsercontext/cookie) into the [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/cookie). All pages within this [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/cookie) will have these [cookies](/javascript-api/k6-experimental/browser/browsercontext/cookie) installed.
```
Comment (ka3de) [inline src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/addCookies.md:6]: Small suggestion:
```suggestion
Adds a list of [cookies](/javascript-api/k6-experimental/browser/browsercontext/cookie) into the [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/cookie). All pages within this [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/cookie) will have these [cookies](/javascript-api/k6-experimental/browser/browsercontext/cookie) set.
```

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/addCookies.md [hunk 3]
```diff
@@ -27,18 +34,47 @@ export const options = {
...
+      // this cookie expires in the past, so it will be removed.
+      {
+        name: 'testcookie3',
+        value: '3',
+        sameSite: 'Lax',
+        domain: '127.0.0.1',
+        path: '/',
+        expires: dayBefore
+      }
+    ]);
```
Comment (ka3de) [inline src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/addCookies.md:74]: Should we do something else with the page so it's a more realistic example? Such as navigating to some page? In this case seems like we are setting cookies for no actual effect.

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/cookies.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+export default async function () {
+  const context = browser.newContext();
+  const page = context.newPage();
+
+  try {
+    // get cookies from the browser context
+    let cookies = context.cookies();
+    console.log("initial cookies length:", cookies.length); // prints 0
+
+    // let's add more cookies to filter by urls.
```
Comment (ka3de) [inline src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/cookies.md:53]: Remove trailing period?
```suggestion
    // let's add more cookies to filter by urls
```

**Review by ka3de — COMMENTED** [review]
Nicely done! :clap: 
Just made some small suggestions.

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/addCookies.md (hunk 3 above)
Comment (inancgumus) [reply src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/addCookies.md:74]: Sure, what do you suggest?

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/cookie.md (hunk 1 above)
Comment (inancgumus) [reply src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/cookie.md]: Sure, see https://github.com/grafana/k6-docs/pull/1328/commits/25091d7a77093b079acf52ee9fe776325867d8f6.

**Review by ka3de — APPROVED** [review]
LGTM :tada:

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext.md [hunk 5]
```diff
@@ -10,17 +10,18 @@ The [browser module API](/javascript-api/k6-experimental/browser#browser-module-
...
-| [BrowserContext.waitForEvent(event[, optionsOrPredicate])](/javascript-api/k6-experimental/browser/browsercontext/waitforevent) <BWIPT id="447"/>           | Waits for the event to fire and passes its value into the predicate function.                              |
+| Method                                                                                                                                                      | Description                                                                                                     |
+| ----------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| [BrowserContext.addCookies()](/javascript-api/k6-experimental/browser/browsercontext/addcookies/)                                                           | Adds cookies into the `BrowserContext`.                                                                         |
```
Comment (ankur22) [inline src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext.md]:
```suggestion
| [BrowserContext.addCookies()](/javascript-api/k6-experimental/browser/browsercontext/addcookies/)                                                           | Adds [cookies](/javascript-api/k6-experimental/browser/browsercontext/cookie) into the `BrowserContext`.                                                                         |
```

WDYT?

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/cookies.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+      options: {
+        browser: {
+            type: 'chromium',
+        },
+      },
+    },
+  },
+};
+
+export default async function () {
```
Comment (ankur22) [inline src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext/cookies.md:44]: Nice example 👏

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext.md [hunk 7]
```diff
@@ -10,18 +10,18 @@ The [browser module API](/javascript-api/k6-experimental/browser#browser-module-
...
 
 
-| Method                                                                                                                                                      | Description                                                                                                     |
-| ----------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
-| [BrowserContext.addCookies()](/javascript-api/k6-experimental/browser/browsercontext/addcookies/)                                                           | Adds cookies into the `BrowserContext`.                                                                         |
-| [BrowserContext.clearCookies()](/javascript-api/k6-experimental/browser/browsercontext/clearcookies/) <BWIPT id="442"/>                                     | Clear the `BrowserContext`'s [cookies](/javascript-api/k6-experimental/browser/browsercontext/cookie).                                                                           |
```
Comment (ankur22) [inline src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext.md]: Nice! 🎉

**Review by ankur22 — APPROVED** [review]
Great work! LGTM 🙂

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext.md (hunk 5 above)
Comment (inancgumus) [reply src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/02 BrowserContext.md]: Thanks for the eagle eyes you have: https://github.com/grafana/k6-docs/pull/1328/commits/c942397ab6b5bf67fd9feef95706f2cad2be64f7

**Review by ankur22 — APPROVED** [review]
LGTM 🚀

**Pull request merged**